src/include/daos/checksum.h

```diff
@@ -159,12 +171,17 @@ daos_csum_type2algo(enum DAOS_CSUM_TYPE type);
  *			for it.
  * @param ft		Pointer to the function table for checksum calculations
  * @param chunk_bytes	Chunksize, typically from the container configuration
+ * @param srv_verify	whether server-side checksum verification is enabled
+ * @param dedup		whether deduplication is enabled on the server
+ * @param dedup_verify	whether to memcmp data on the server for deduplication
+ * @param dedup_bytes	deduplication size threashold in bytes
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'threashold' may be misspelled - perhaps 'threshold'?